libr/asm/asm.c

```diff
@@ -309,10 +309,12 @@ R_API int r_asm_use(RAsm *a, const char *name) {
 	r_list_foreach (a->plugins, iter, h) {
 		if (!strcmp (h->name, name) && h->arch) {
 			if (!a->cur || (a->cur && strcmp (a->cur->arch, h->arch))) {
-				char *file = r_str_newf ("%s/%s.sdb", r_str_r2_prefix (R2_SDB_OPCODES), h->arch);
+				char *r2prefix = r_str_r2_prefix (R2_SDB_OPCODES);
+				char *file = r_str_newf ("%s/%s.sdb", r2prefix, h->arch);
```

Comment:
r2prefix can be null, so better use r_str_get in here to avoid the undefined behaviour of passing NULL to %s. or just if (r2prefix) { ...